web/modules/custom/user_registration/images/drupal-evergreen-logo.svg

```diff
@@ -0,0 +1,12 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 42.2 55.5">
```

Comment:
As we noticed, we'd need to tweak this to full numbers like: `viewBox="0 0 44 58"`

Reply:
Thanks! Oddly here it does not seem to have any influence, perhaps it was related to the path.
![Screenshot 2020-05-24 at 01 22 36](https://user-images.githubusercontent.com/525003/82742349-2bc3e800-9d5d-11ea-8f8f-5face1cf7a93.png)

Reply:
Still rounded the viewBox https://github.com/drupal-celebrations/celebrate-drupal-9/pull/29/commits/ec5017984e6c334ff9c3450c8894f9d770267088